src/libostree/ostree-repo.c

```diff
@@ -2653,6 +2653,37 @@ get_remotes_d_dir (OstreeRepo          *self,
     return g_file_resolve_relative_path (sysroot, SYSCONF_REMOTES);
 }
 
+static gboolean
+min_free_space_calculate_reserved_bytes (OstreeRepo *self, guint64 *bytes, GError **error)
+{
+  guint64 reserved_bytes = 0;
+
+  struct statvfs stvfsbuf;
+  if (TEMP_FAILURE_RETRY (fstatvfs (self->repo_dir_fd, &stvfsbuf)) < 0)
+    return glnx_throw_errno_prefix (error, "fstatvfs");
```

Comment:
I would be tempted to set `*bytes = 0` when doing an error return, but it’s not strictly necessary and it’s more a matter of whether that’s part of OSTree policy than anything else. My thinking is that it’s safer to set it to a value than leave it uninitialised and potentially have the caller read it and do something silly (even though they’re not supposed to read it on error).

Reply:
Yeah, we don't normally do so this is fine as is to me. Though I wouldn't oppose it either.

Reply:
I’ll merge then, unchanged. Thanks for your response!